internal/tools/utility/envvariable/setenvvariable.go

```diff
@@ -0,0 +1,209 @@
+	}
+
+	t := Tool{
+		Name:        cfg.Name,
+		Kind:        kind,
+		Parameters:  parameters,
+		manifest:    tools.Manifest{Description: cfg.Description, Parameters: parameters.Manifest(), AuthRequired: cfg.AuthRequired},
+		mcpManifest: mcpManifest,
+	}
+	return t, nil
+}
+
+// validate interface
+var _ tools.Tool = Tool{}
+
+type Tool struct {
+	Name        string
+	Kind        string
+	Parameters  tools.Parameters
+	manifest    tools.Manifest
+	mcpManifest tools.McpManifest
+}
+
+func (t Tool) Invoke(ctx context.Context, params tools.ParamValues) ([]any, error) {
+	paramsMap := params.AsMap()
+	mcpSettingsFile, ok := paramsMap["mcpSettingsFile"]
+	if !ok {
+		return nil, fmt.Errorf("mcpSettingsFile not found in params")
+	}
+
+	mcpSettingsFileStr, ok := mcpSettingsFile.(string)
+	if !ok {
+		return nil, fmt.Errorf("mcpSettingsFile is not a string")
+	}
+
+	data, err := os.ReadFile(mcpSettingsFileStr)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read mcp settings file: %w", err)
+	}
+
+	var mcpSettings map[string]interface{}
+	if err := json.Unmarshal(data, &mcpSettings); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal mcp settings file: %w", err)
+	}
+
+	mcpServers, ok := mcpSettings["mcpServers"].(map[string]interface{})
+	if !ok {
+		if servers, found := mcpSettings["servers"].(map[string]interface{}); found {
+			mcpServers = servers
+		} else {
+			mcpServers = make(map[string]interface{})
+			mcpSettings["mcpServers"] = mcpServers
+		}
+	}
+
+	var targetServer map[string]interface{}
+	var targetServerName string
+	for serverName, server := range mcpServers {
+		serverMap, ok := server.(map[string]interface{})
+		if !ok {
+			continue
+		}
+		args, ok := serverMap["args"].([]interface{})
+		if !ok {
+			continue
+		}
+		for _, arg := range args {
+			if argStr, ok := arg.(string); ok && argStr == "alloydb-postgres" {
+				targetServer = serverMap
+				targetServerName = serverName
+				break
+			}
+		}
+		if targetServer != nil {
+			break
+		}
+	}
+
+	if targetServer == nil {
+		targetServerName = "alloydb"
+		targetServer = make(map[string]interface{})
+		targetServer["args"] = []interface{}{"--prebuilt", "alloydb-postgres", "--stdio"}
+		mcpServers[targetServerName] = targetServer
+	}
+
+	if _, ok := targetServer["command"]; !ok {
+		targetServer["command"] = "./PATH/TO/toolbox"
+	}
+
+	env, ok := targetServer["env"].(map[string]interface{})
+	if !ok {
+		env = make(map[string]interface{})
+		targetServer["env"] = env
+	}
+
+	for key, value := range paramsMap {
+		if key != "mcpSettingsFile" {
+			env[key] = value
+		}
+	}
+
+	updatedData, err := json.MarshalIndent(mcpSettings, "", "  ")
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal mcp settings file: %w", err)
+	}
+
+	if err := os.WriteFile(mcpSettingsFileStr, updatedData, 0644); err != nil {
+		return nil, fmt.Errorf("failed to write mcp settings file: %w", err)
+	}
+
+	return []any{"Successfully updated MCP settings file"}, nil
```

Comment:
```suggestion
	return "Successfully updated MCP settings file", nil
```

Just incase... This change is only applicable if this PR is merge after #904 is merged. The return type will also be updated to `any`.